.buildkite/scripts/test_basic_cluster.sh

```diff
@@ -48,4 +48,4 @@ echo "producing to topic"
 echo test | "${PATH_TO_RPK_FILE}" topic produce testtopic --brokers "$REDPANDA_BROKERS" -v || exit 1
 
 echo "consuming from topic"
-"${PATH_TO_RPK_FILE}" topic consume testtopic --brokers "$REDPANDA_BROKERS" -v -o :end | grep -q test || exit 1
+"${PATH_TO_RPK_FILE}" topic consume testtopic --brokers "$REDPANDA_BROKERS" -v -o :end | grep test || exit 1
```

Comment:
[not to change] I thought you changed this to grep for 'squirrel'

Reply:
That was in the scrapped PR. I caught most of the stuff but missed this one 😩 

Will get in next PR